### PR TITLE
remove optionnel word in modification load form

### PR DESCRIPTION
**Pull request by Walid-Sahnoun**

### CODE REVIEW

File: src/components/refactor/dialogs/load-modification/load-modification-dialog.js [hunk 1]
```diff
@@ -120,7 +120,11 @@ const LoadModificationDialog = ({
     }, [reset, emptyFormData]);
 
     return (
-        <FormProvider validationSchema={schema} {...methods}>
+        <FormProvider
+            validationSchema={schema}
+            {...methods}
+            isHypothesisEditing={editData ? true : false}
```
[the remark this answers is not in the document]
Comment (Walid-Sahnoun) [reply src/components/refactor/dialogs/load-modification/load-modification-dialog.js]: done

File: src/components/refactor/dialogs/load-modification/load-modification-dialog.js [hunk 2]
```diff
@@ -120,7 +120,11 @@ const LoadModificationDialog = ({
     }, [reset, emptyFormData]);
 
     return (
-        <FormProvider validationSchema={schema} {...methods}>
+        <FormProvider
+            validationSchema={schema}
+            {...methods}
+            removeOptionnel={true}
```
Comment (sBouzols) [inline src/components/refactor/dialogs/load-modification/load-modification-dialog.js:126]: `optional` not `optionnel` which is french
And by the way do we want to pass our props through FormProvider ? 
This is strange

Comment (sBouzols) [reply src/components/refactor/dialogs/load-modification/load-modification-dialog.js:126]: Ok `validationSchema` is also our prop, maybe move its declaration befor `{...methods}` order is important here.

Comment (Walid-Sahnoun) [reply src/components/refactor/dialogs/load-modification/load-modification-dialog.js]: done

**Review by EstherDarkish — APPROVED** [review]

**Pull request merged**